src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Default {
```

Comment:
The class `RightTriangle` should not extend `Default` if `Default` is an abstract class used to set behavior. Consider using an interface for behavior definition if applicable.